web/src/engine/websites/XCaliBRScans_e2e.ts

```diff
@@ -1,7 +1,55 @@
 import { describe } from 'vitest';
 import { TestFixture, type Config } from '../../../test/WebsitesFixture';
 
-const config: Config = {
+const configNormal: Config = {
+    plugin: {
+        id: 'xcalibrscans',
+        title: 'xCaliBR Scans'
+    },
+    container: {
+        url: 'https://xcalibrscans.com/webcomics/manga/the-first-ancestor-in-history/',
+        id: '/webcomics/manga/the-first-ancestor-in-history/',
+        title: 'The First Ancestor in History'
+    },
+    child: {
+        id: '/the-first-ancestor-in-history-chapter-1/',
+        title: 'Chapter 1',
+    },
+    entry: {
+        index: 0,
+        size: 771_615,
+        type: 'image/jpeg'
+    }
+};
+
+const fixtureNormal = new TestFixture(configNormal);
+describe(fixtureNormal.Name, async () => (await fixtureNormal.Connect()).AssertWebsite());
+
+const configScrambled1: Config = {
+    plugin: {
+        id: 'xcalibrscans',
+        title: 'xCaliBR Scans'
+    },
+    container: {
+        url: 'https://xcalibrscans.com/webcomics/manga/the-first-ancestor-in-history/',
+        id: '/webcomics/manga/the-first-ancestor-in-history/',
+        title: 'The First Ancestor in History'
+    },
+    child: {
+        id: '/the-first-ancestor-in-history-chapter-86/',
+        title: 'Chapter 86',
+    },
+    entry: {
+        index: 0,
+        size: 5_250_563,
+        type: 'image/png'
+    }
+};
+
+const fixtureScrambled1 = new TestFixture(configScrambled1);
```

Comment:
Avoid using counters for variable names. Use ore expressive name to describe the variable.